pkg/networkservice/connectioncontext/client.go

```diff
@@ -17,9 +17,10 @@
 package connectioncontext
 
 import (
-	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
 	"github.com/networkservicemesh/sdk/pkg/networkservice/core/chain"
 
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
```

Comment:
I'm curious why these header orders changed here and in server.go.  Not going to hold this PR up over it, but it suggests perhaps the goimports linting isn't strict enough.

Reply:
Probably it is a problem of configuring `golangci-linter`.  I can take a look at this issue in separate PR if you don't mind.

Reply:
Definitely shouldn't hold up this PR.